vizro-core/examples/visual-vocabulary/pages/examples/sparkline.py

```diff
@@ -0,0 +1,17 @@
+import plotly.express as px
+from vizro.models.types import capture
+
+stocks = px.data.stocks()
+stocks_melt = stocks.melt(id_vars="date", value_vars=["GOOG", "AMZN", "AAPL"], var_name="stock", value_name="value")
+
+
+@capture("graph")
+def sparkline(data_frame, **kwargs):
+    fig = px.line(data_frame, **kwargs)
+    fig.update_xaxes(visible=False)
+    fig.update_yaxes(visible=False)
+    fig.update_layout(showlegend=False)
+    return fig
+
+
```

Comment:
With other charts (butterfly, dumbbells, diverging stacked bar) we have kept the data in wide form to avoid doing `melt` so I would favour doing so here as well unless you disagree strongly.

These two are identical but I prefer the first since it doesn't need any knowledge of "advanced" pandas. Internally when plotting wideform data plotly does exactly the `melt` operation you do manually here so it's kind of unnecessary for us to do it.

```
fig = sparkline(
    stocks, x="date", y=["GOOG", "AMZN", "AAPL"], labels={"variable": "stock"}, facet_row="variable"
)

fig2 = sparkline(stocks_melt, x="date", y="value", color="stock", facet_row="stock")
```